policy_set_integration_test.go

```diff
@@ -86,6 +88,19 @@ func TestPolicySetsList(t *testing.T) {
 		assert.Equal(t, workspace.ID, psl.Items[0].Workspaces[0].ID)
 	})
 
+	t.Run("with workspace exclusion include param", func(t *testing.T) {
+		skipUnlessBeta(t)
+		psl, err := client.PolicySets.List(ctx, orgTest.Name, &PolicySetListOptions{
+			Include: []PolicySetIncludeOpt{PolicySetWorkspaceExclusions},
+		})
+		require.NoError(t, err)
+		assert.Equal(t, 2, len(psl.Items))
+
+		assert.NotNil(t, psl.Items[0].WorkspaceExclusions)
+		assert.Equal(t, 1, len(psl.Items[0].WorkspaceExclusions))
+		assert.Equal(t, excludedWorkspace.ID, psl.Items[0].WorkspaceExclusions[0].ID)
```

Comment:
One minor issue with this test:

If there are workspace exlusions, the ID is always returned. The Include parameter on the list options embeds the entire workspace into the struct. Can you test that you have some extra data, like the name? See [this example](https://github.com/hashicorp/go-tfe/blob/main/policy_set_integration_test.go#L417-L433)

Reply:
Added Name and createdAt property.